plugins/decl_hdf5/dataset_op.cxx

```diff
@@ -203,8 +211,12 @@ void Dataset_op::fletcher(Context& ctx, Expression value)
 	}
 }
 
-void Dataset_op::execute(Context& ctx, hid_t h5_file, hid_t xfer_lst, const unordered_map<string, Datatype_template_sptr>& dsets)
+void Dataset_op::execute(Context& ctx, hid_t h5_file, bool use_mpio, const unordered_map<string, Datatype_template_sptr>& dsets)
 {
+	Raii_hid xfer_lst = make_raii_hid(H5Pcreate(H5P_DATASET_XFER), H5Pclose);
+	if (use_mpio) {
+		if (0 > H5Pset_dxpl_mpio(xfer_lst, m_mpio)) handle_hdf5_err();
```

Comment:
My personal opinion: I think that all if/else blocks should use curly braces, even if there is only 1 line.

```suggestion
		if (0 > H5Pset_dxpl_mpio(xfer_lst, m_mpio)) {
			handle_hdf5_err();
		}
```